test/fields.uts

```diff
@@ -27,12 +27,15 @@ assert LELongField("test", None).addfield(None, b"", 0x44434241) == b'ABCD\x00\x
 = MACField class
 ~ core field
 m = MACField("foo", None)
-m.i2m(None, None)
-assert( _ == b"\x00\x00\x00\x00\x00\x00" )
-m.getfield(None, b"\xc0\x01\xbe\xef\xba\xbeABCD")
-assert( _ == (b"ABCD","c0:01:be:ef:ba:be") )
-m.addfield(None, b"FOO", "c0:01:be:ef:ba:be")
-assert( _ == b"FOO\xc0\x01\xbe\xef\xba\xbe" )
+r = m.i2m(None, None)
+r
+assert(r == b"\x00\x00\x00\x00\x00\x00" )
```

Comment:
You don't nee parenthesis in `assert` statements (you have a lot of those).

Reply:
Done, and the other ones `assert`.